components/email-form/EmailFormContainer.tsx

```diff
@@ -42,10 +42,9 @@ const InputSection = () => {
     e.preventDefault();
     if (emailService.isValid(email)) {
       emailService.subscribe(email)
-        .then(success => {
-          if (success) {
-            setSubscribeMessage("Thank you for subscribing.");
-          }
+        .then(() => {
+          setSubscribeMessage("Thank you for subscribing.");
+          setEmail('');
```

Comment:
Just making sure we mean to rest the email to ''

Reply:
Intentional.